app/views/tags/aggregatedMetrics.scala.html

```diff
@@ -17,10 +17,7 @@
 @(result: models.AppResult)
 @import com.linkedin.drelephant.util.Utils;
 
-@*
-* Spark is currently not supported for this feature. It will be added in the future.
```

Comment:
It would be better not to show the delay for Spark jobs rather than showing 0%.

We can retain this condition below for delay.